README.md

```diff
@@ -116,32 +116,76 @@ On OS X:
-tar -xvf solr-4.10.4.tgz
-sudo cp -R solr-4.10.4/example /opt/solr
+NYC Councilmatic uses Solr version 7 to power search fro legislation. If you already have Solr version 7 installed for another app, then you can by-pass these installation instructions. Otherwise, carefully read on.
+
+Download latest solr distribution from a [reliable mirror](http://www.apache.org/dyn/closer.cgi/lucene/solr/). The Apache foundation suggests using the [University of Toronto mirror](http://mirror.dsrg.utoronto.ca/apache/lucene/solr/). You need to download two files:
+(1) `solr-7.1.0.tgz` - Solr itself
+(2) solr-7.1.0.tgz.asc - the detached signature for verification. (If you do not find a signature in the Toronto mirror, then grab a signature from [elsewhere](http://www.apache.org/dist/lucene/solr/7.1.0/). You can `curl` or `wget` the signature to the directory where you downloaded Solr.
+
+Next, [verify the package](https://www.apache.org/info/verification.html) using the .asc signature from the official solr mirror. Apache provides some rigid standards for verification, e.g., "face-to-face communication with multiple government-issued photo identification confirmations." However, you can also conduct your own Google background check on the signature owner to validate their identity. 
+
+Finally, untar the directory (`tar xvf solr-7.1.0.tgz`), and `mv` it into `/opt` (or `/Applications`, for Mac users).
+
+Congratulations! You downloaded Solr.
+
+**Setup Solr**
+
+The next steps are fairly straightforward:
+
+1. **Visit** your latest download, take a look at the solr directory (`cd solr-7.1.0/server/solr`), and within it, create a repo for NYC Councilmatic: `mkdir nyc-council-councilmatic`
+2. In the newly created nyc-council-councilmatic repo, **create** a core file: `touch nyc-council-councilmatic/core.properties` (This file helps solr discover cores for multicore processing. That way, one Solr installation can run multiple apps.)
+3. Solr expects to find several files in the conf repo of Councilmatic. We'll use the example conf (`sample_techproducts_configs`) and parse it down. **Copy** the contents of `solr/configsets/sample_techproducts_configs/` into `solr/nyc-council-councilmatic`: `cp -R configsets/sample_techproducts_configs/* nyc-council-councilmatic/`.
+
+Note: you do not need all the files provided in `sample_techproducts_configs`. You can safely remove several, including: _rest_managed.json, _schema_analysis_stopwords_english.json, _schema_analysis_synonyms_english.json, mapping-FoldToASCII.txt, mapping-ISOLatin1Accent.txt, update-script.js.
+
+Next, you need to set-up the NYC Councilmatic schema. We use a [classic schema file](https://lucene.apache.org/solr/guide/7_1/schema-factory-definition-in-solrconfig.html#switching-from-managed-schema-to-manually-edited-schema-xml). First, open `solrconfig.xml`, which lives inside `/solr/nyc-council-councilmatic/conf/`. Anywhere between the `<config>` tags, add the following:
```

Comment:
Seems likely we'll want to eventually start using the newer style schema. But that's probably a task for a later date.

Reply:
Yes, I agree...